src/js/core/range.js

```diff
@@ -605,7 +605,25 @@ define([
        * @return {WrappedRange}
        */
       createFromNode: function (node) {
-        return this.create(node, 0, node, 1);
+        var sc = node;
+        var so = 0;
+        var ec = node;
+        var eo = dom.nodeLength(ec);
+
+        // browsers can't target a picture or void node
+        if (dom.isVoid(sc)) {
+          so = dom.listPrev(sc).length - 1;
+          sc = sc.parentNode;
+        }
+        if (dom.isBR(ec)) {
+          eo = dom.listPrev(ec).length - 1;
+          ec = ec.parentNode;
+        } else if (dom.isVoid(ec) || dom.isImg(sc)) {
```

Comment:
@Gorash 
is this condition should be `dom.isVoid(ec) || dom.isImg(ec)`(not `sc`)? If it is, We can handle this with only `dom.isVoid(ec)`.